src/main/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAge.java

```diff
@@ -54,6 +56,12 @@ protected boolean verifyKey(String key) {
     }
 
     protected int getMaxAge(HttpServletRequest request) throws Exception {
+        if (appConfig().babbage().isLegacyCacheAPIEnabled()) {
+            LegacyProxyMaxAge legacyProxyMaxAge = new LegacyProxyMaxAge();
```

Comment:
Same as before, you should rename all the usages of Legacy Proxy to Legacy Cache Proxy, for consistency and clarity. This happens in several other places.